src/current/cockroachcloud/upgrade-to-v23.2.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Upgrade to CockroachDB v23.2 Pre-Production Preview
+summary: Learn how to upgrade your CockroachDB cluster to v23.2 Pre-Production Preview
+toc: true
+docs_area: manage
+page_version: v23.2
+prev_version: v23.1
+pre_production_preview: True
+pre_production_preview_version: v23.2.0-beta.3
+---
+
+{% if page.pre_production_preview == true %}
+[CockroachDB {{ page.pre_production_preview_version }}](https://www.cockroachlabs.com/docs/releases/{{ page.page_version }}#{{ page.pre_production_preview_version | replace: ".","-"}}) is available to CockroachDB {{ site.data.products.dedicated }} clusters for testing and experimentation.
+
+{{site.data.alerts.callout_danger}}
+This [testing release]({% link releases/index.md %}#release-naming) is not qualified for production environments and not eligible for support or uptime SLA commitments.
+{{site.data.alerts.end}}
+
+An [Org Administrator]({% link cockroachcloud/authorization.md %}#org-administrator-legacy) can upgrade your CockroachDB {{ site.data.products.dedicated }} cluster from the CockroachDB {{ site.data.products.cloud }} Console. This page guides you through the process of upgrading.
+
+{{site.data.alerts.callout_success}}
+Upgrading from {{ page.prev_version }} to {{ page.pre_production_preview_version }} is a major-version upgrade. Upgrading a CockroachDB {{ site.data.products.dedicated }} cluster to a new major version is opt-in. Before proceeding, review the CockroachDB {{ site.data.products.cloud }} [upgrade policy](https://cockroachlabs.com/docs/cockroachcloud/upgrade-policy#pre-production-preview). After a cluster is upgraded to a Pre-Production Preview build, subsequent patch releases are applied automatically, including additional v{{ page.page_version }} Beta and Release Candidate (RC) releases, the GA release, and patch releases after GA.
```

Comment:
Edited because the existing text ("subsequent patch releases [...], including [...]) makes it sound like the later RC/beta releases _are_ patch releases, but that contradicts our definition of patch releases on another page.

```suggestion
Upgrading from {{ page.prev_version }} to {{ page.pre_production_preview_version }} is a major-version upgrade. Upgrading a CockroachDB {{ site.data.products.dedicated }} cluster to a new major version is opt-in. Before proceeding, review the CockroachDB {{ site.data.products.cloud }} [upgrade policy](https://cockroachlabs.com/docs/cockroachcloud/upgrade-policy#pre-production-preview). After a cluster is upgraded to a Pre-Production Preview release, it is automatically upgraded to all subsequent releases within v{{ page.page_version }}—including additional beta and RC releases, the GA release, and patch releases after GA—which are all treated as [patch version upgrades]({% link cockroachcloud/upgrade-policy.md %}#patch-version-upgrades).
```

Reply:
Reworked to avoid the need for an emdash.